pandas/core/generic.py

```diff
@@ -5538,6 +5538,24 @@ def astype(
         0    10
         1     2
         dtype: int64
+
+        Create a series of dates:
+
+        >>> ser_date = pd.Series(pd.date_range('20200101', periods=3))
+        >>> ser_date
+        0   2020-01-01
+        1   2020-01-02
+        2   2020-01-03
+        dtype: datetime64[ns]
+
+        Convert to datetime type with time zone:
+
+        >>> # localize to UTC and convert to US/Eastern time zone
```

Comment:
Instead of the comment here, can you add it as commentary with `Convert to datetime type with time zone:`

Reply:
Sure @mroeschke

I'll make that change and update the PR.